### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Computability/AkraBazzi/AkraBazzi.lean [hunk 1]
```diff
@@ -427,7 +427,7 @@ lemma eventually_one_add_smoothingFn_pos : ∀ᶠ (n:ℕ) in atTop, 0 < 1 + ε n
   refine Eventually.nat_cast_atTop (p := fun n => 0 < 1 + ε n) ?_
   filter_upwards [h₁ (by norm_num : (0:ℝ) < 1/2), eventually_gt_atTop 1] with x _ hx'
   have : 0 < log x := Real.log_pos hx'
-  show 0 < 1 + 1 / log x
+  unfold smoothingFn
```
Comment (jcommelin) [inline Mathlib/Computability/AkraBazzi/AkraBazzi.lean:430]: The version with `show` is more readable. So I think we should keep it.

File: Mathlib/Computability/AkraBazzi/AkraBazzi.lean [hunk 2]
```diff
@@ -438,12 +438,11 @@ lemma eventually_one_add_smoothingFn_nonneg : ∀ᶠ (n:ℕ) in atTop, 0 ≤ 1 +
   filter_upwards [eventually_one_add_smoothingFn_pos] with n hn; exact le_of_lt hn
 
 lemma strictAntiOn_smoothingFn : StrictAntiOn ε (Set.Ioi 1) := by
-  show StrictAntiOn (fun x => 1 / log x) (Set.Ioi 1)
+  unfold smoothingFn
```
Comment (jcommelin) [inline Mathlib/Computability/AkraBazzi/AkraBazzi.lean:441]: Idem

**Review by jcommelin — COMMENTED** [review]
bors d+

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**